docs/docker/redirects.conf

```diff
@@ -64,10 +64,14 @@ rewrite ^/((\d+\.\d+|next)/)?tutorial-performance-tips/?$ /docs/$1performance/ p
 rewrite ^/((\d+\.\d+|next)/)?tutorial-release-notes/?$ /docs/$1release-notes/ permanent;
 rewrite ^/((\d+\.\d+|next)/)?tutorial-migration-guide/?$ /docs/$1migration-from-7.4-to-8.0/ permanent;
 rewrite ^/((\d+\.\d+|next)/)?tutorial-known-limitations/?$ /docs/$1third-party-licenses/ permanent;
-rewrite ^/((\d+\.\d+|next)/)?keyboard-navigation/?$ /docs/$1keyboard-shortcuts/ permanent;
-rewrite ^/((\d+\.\d+|next)/)?internationalization-i18n/?$ /docs/$1language/ permanent;
+rewrite ^/keyboard-navigation/?$ /docs/keyboard-shortcuts/ permanent;
+rewrite ^/(12\.0|next)/keyboard-navigation/?$ /docs/$1/keyboard-shortcuts/ permanent;
+rewrite ^/(11\.1|11\.0|10\.0|9\.0)/keyboard-shortcuts/?$ /docs/$1/keyboard-navigation/ permanent;
+rewrite ^/internationalization-i18n/?$ /docs/language/ permanent;
+rewrite ^/(12\.0|next)/internationalization-i18n/?$ /docs/$1/language/ permanent;
+rewrite ^/(11\.1|11\.0|10\.0|9\.0)/language/?$ /docs/$1/internationalization-i18n/ permanent;
```

Comment:
The above rewrites do not match the table in the PR description. 

Also, how did you test it? It seems that the `$1` is inserted wrongly, comparing to other rules e.g. on line 66.

Reply:
https://github.com/handsontable/handsontable/pull/9453#issuecomment-1119386789